ReleaseTooling/Sources/PodspecsTester/main.swift

```diff
@@ -62,8 +62,9 @@ struct PodspecsTester: ParsableCommand {
         return "--\(key)"
       }
     }.joined(separator: " ")
+    // TODO(andrewheard): Remove SpecsDev URL from sources before merging into `master`.
     let command =
-      "pod spec lint \(spec) \(arguments) --sources=https://github.com/firebase/SpecsTesting,https://cdn.cocoapods.org/"
+      "pod spec lint \(spec) \(arguments) --sources=https://github.com/firebase/SpecsTesting,https://github.com/firebase/SpecsDev.git,https://cdn.cocoapods.org/"
```

Comment:
Delete this and move AppCheckCore to SpecsStaging

Reply:
I staged https://github.com/firebase/SpecsStaging/blob/2b63cf19d99a309f6bf1c40335405ce08e33344a/AppCheckCore/10.18.0/AppCheckCore.podspec and changed `SpecsDev` to `SpecsStaging` because `AppCheckCore` isn't in `SpecsTesting` ([error](https://github.com/firebase/firebase-ios-sdk/actions/runs/6725525769/job/18280150823?pr=12030)).